quickwit-config/src/config.rs

```diff
@@ -483,8 +483,10 @@ impl QuickwitConfig {
             .with_port(rest_listen_port)
             .to_socket_addr()
             .expect("The default host should be an IP address.");
+        let grpc_listen_port = quickwit_common::net::find_available_tcp_port()
```

Comment:
Note: when calling consecutively `find_available_tcp_port`, the OS increments by one the port and if you start several nodes consecutively, there will be an overlap between grpc and rest ports.